CONTRIBUTING.md

```diff
@@ -0,0 +1,163 @@
+   sure the issue hasn't already been reported. If you find a similar bug and the issue is still open, consider adding
+   a comment providing any new information that you might be able to report.
+2. Make sure the issue hasn't been fixed already. Try to reproduce it using the latest `main` branch in the repository if
+   you were not working with the latest version.
+3. Make sure the bug is really a bug. If you need general support or are unsure whether some behaviour represents a bug
+   please do not file a bug ticket but reach out through the above mentioned [communication](#communication) channels.
+4. Gather as much information about the bug as you can. Logs, screenshots/screen captures, steps to reproduce the bug
+   can be vital for a useful bug report.
+
+If you already have suggestions on how to fix the bug, do not hesitate to include them in the bug description.
+
+## Feature requests
+
+Just like bugs, feature requests are tracked as [GitHub issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues).
+When suggesting an enhancement choose "Feature Request" when [creating](https://github.com/teamhanko/hanko/issues/new/choose) a new
+issue and fill out the template form.
+
+Before making a feature request:
+
+1. Take a look at the [existing issues](https://github.com/teamhanko/hanko/issues?q=is%3Aissue+is%3Aopen+label%3Aenhancement) and make
+   sure the issue hasn't already been reported.
+2. Make sure the issue hasn't been implemented already. Always try using the latest `main` branch in the repository if
+   to confirm the feature is not already in place.
+
+When filling out the template form, be sure to be as detailed as possible. Describe the current behavior and explain
+which behavior you expect to see instead. Explain why this enhancement would be useful to Hanko users.
+
+## Submitting Code
+
+Contributing code requires a [GitHub](https://github.com/) account. All contributions are made via
+[pull requests](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests).
+Pull requests should target the `main` branch.
+
+To submit your code:
+
+1. [Fork](https://docs.github.com/en/get-started/quickstart/fork-a-repo#forking-a-repository) the [repository](https://github.com/teamhanko/hanko).
+2. [Clone](https://docs.github.com/en/get-started/quickstart/fork-a-repo#cloning-your-forked-repository) the forked repository.
+3. [Configure remotes](https://docs.github.com/en/get-started/quickstart/fork-a-repo#configuring-git-to-sync-your-fork-with-the-original-repository).
+4. Create a feature [branch](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-branches)
+   off of the `main` branch.
+   ```
+   git checkout -b <feature-branch-name>
+   ```
```

Comment:
Is the branch still a `feature branch`, when you want to fix a bug? I think it should just say "Create a new branch ..." and instead of "feature-branch-name", "new-branch-name" or so.